libvast/include/vast/plugin.hpp

```diff
@@ -333,6 +340,32 @@ class query_language_plugin : public virtual plugin {
   parse(std::string_view query) const = 0;
 };
 
+// -- rest endpoint plugin -----------------------------------------------------
+
+class rest_endpoint_plugin : public virtual plugin {
```

Comment:
Naming question: it's the REST endpoint plugin, the REST handler actor, but a set of API endpoint. Can you add a comment on how they relate, ideally somewhere in the plugin API?